Source/Bricks/Collection/CollectionBrick.swift

```diff
@@ -9,17 +9,22 @@
 import UIKit
 
 public typealias ConfigureCollectionBrickBlock = ((cell: CollectionBrickCell) -> Void)
+public typealias RegisterBricksCollectionBrickBlock = ((cell: CollectionBrickCell) -> Void)
 
 // MARK: - Brick
 
 public class CollectionBrick: Brick {
     let dataSource: CollectionBrickCellDataSource
     let scrollDirection: UICollectionViewScrollDirection
     var shouldCalculateFullHeight: Bool = true // This flag indicates that the collection brick is small enough to calculate its whole height directly
-
-    public init(_ identifier: String = "", width: BrickDimension = .Ratio(ratio: 1), height: BrickDimension = .Auto(estimate: .Fixed(size: 50)), backgroundColor: UIColor = .clearColor(), backgroundView: UIView? = nil, scrollDirection: UICollectionViewScrollDirection = .Vertical, dataSource: CollectionBrickCellDataSource) {
+    var brickTypes = [Brick.Type]()
```

Comment:
No need to give this an empty array as value. It will be forced by the initializer. And because you added a default implementation, it will either be an empty array or the given value